### PR TITLE
Rename table

### PR DESCRIPTION
**Pull request by billy-addepar**
Rename `ember-table-2` -> `ember-table`

### CODE REVIEW

**Comment by billy-addepar** [comment]
@pzuraq

**Comment by pzuraq** [comment]
Let's not disable travis and fix the tests now, we should be running them on all versions

**Comment by billy-addepar** [comment]
Given that it takes time to fix the `ember-try` and publish Addepar stylegiude, I think we should go ahead and merge this PR.

**Comment by pzuraq** [comment]
I'm ok with it if we remove the changes to Ember try config, if tests are going to fail anyways we shouldn't mess with the test suite.

File: LICENSE.md [hunk 1]
```diff
@@ -1,9 +1,25 @@
-The MIT License (MIT)
+Copyright © 2017 Addepar, Inc. All Rights Reserved.
```
Comment (billy-addepar) [inline LICENSE.md]: This license is the same as the old one:

https://github.com/Addepar/ember-table/blob/addepar/LICENSE.md

Only the year is changed.

**Comment by billy-addepar** [comment]
Yay, tests pass. I don't why it passes but it passes. Ready for review again

File: .travis.yml [hunk 2]
```diff
@@ -13,11 +13,6 @@ cache:
 env:
   # we recommend new addons test the current and previous LTS
   # as well as latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-lts-2.8
```
Comment (pzuraq) [inline .travis.yml]: We still need to revert these changes, and we really should add `ember-1.11` too but we can get this merged for now

**Comment by billy-addepar** [comment]
@pzuraq alright, I added a number of test scenarios in the travis file, including 1.13, 2.8, etc. All of them pass now. 1.11 is failing `You can only pass attributes (such as name=value) not bare values to a helper for a View found in` and I have not figure out how to fix that in test. But at least we have coverage for a number of Ember versions.

**Comment by pzuraq** [comment]
Ah, to fix that just add `import 'vertical-collection/-debug';` to `test-helper.js`

**Comment by billy-addepar** [comment]
Adding `import 'vertical-collection/-debug';` does not solve it. I still get the same issue

**Comment by billy-addepar** [comment]
Yay, all tests pass including 1.11
